### DISCUSSION TITLE
非必要情况不要开放注册

### DISCUSSION THREAD

**Discussion by KKISO574** [lsky-org/lsky-pro#665] [分享]
刚刚数据库被删了，网站目录没有其他木马，这个傻逼还在数据库里面留了一个btc地址，估计通过图床上传图片的木马进行删库的，一定要注意

**Comment by 0xxb** [comment]
麻烦描述一下具体细节，方便我来排查原因。
- 服务器型号及版本
- Nginx 或 Apache 和 PHP 版本
- 服务器是否存在其他网站
- 使用了什么类型的数据库
- 数据库是否使用了弱口令
- 数据库是否允许了外网访问权限
- Lsky Pro 版本
- 使用的是什么储层策略

谢谢！

**Reply by KKISO574** [reply]
阿里云服务器，Debian11内核.6.1.0
NGINX1.8
PHP8.0.26
服务器用于个人博客，图片数据存储
mysql5.7.37
密码没有弱口令，使用大小写加@以及数字
数据库端口是开放的
Lsky开源版2.1
存储策略是一个本地一个阿里oss，阿里oss其他用户无法设置的是无法使用。
